Add C++ to problem (#2413)
cpinitiative/usaco-guide: solutions/usaco-915.mdx

```

</JavaSection>
<CPPSection>
	
```cpp
#include <iostream>
#include <algorithm>
#include <vector>

using namespace std;

int main() {
	freopen("herding.in", "r", stdin);
	freopen("herding.out", "w", stdout);
	
	// all cow locations
	vector<int> a;
	for (int i = 0; i < 3; i++) {
		int b;
		cin >> b;
		a.push_back(b);
	}
	sort(a.begin(), a.end());

> 	// min number of moves can either be 0, 1, or 2. 0 only if they are already consecutive, 
> 	// 1 if there is a difference of 2 b/w any two numbers, and 2 in all other cases.

Reviewer: ```suggestion
	/*
	 * The minimum number of moves can only be 0, 1, or 2.
	 * 0 is if they're already consecutive,
	 * 1 is if there's a difference of 2 between any 2 numbers,
	 * and 2 is for all other cases.
	 */
```
